### PR TITLE
[zh-cn]: create the translation of CSS `::-moz-list-number` pseudo-element

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/css/_doublecolon_-moz-list-number/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: "::-moz-list-number"
+slug: Web/CSS/::-moz-list-number
+l10n:
+  sourceCommit: c8b447485fd893d5511d88f592f5f3aec29a725b
+---
+
+{{CSSRef}}{{Non-standard_header}}
+
+**`::-moz-list-number`** [CSS](/zh-CN/docs/Web/CSS) [伪元素](/zh-CN/docs/Web/CSS/Pseudo-elements)是 [Mozilla 扩展](/zh-CN/docs/Web/CSS/Mozilla_Extensions)，表示有序列表（{{HTMLElement("ol")}}）中列表项（{{HTMLElement("li")}}）的标记（通常是数字）。
```
Comment (T34-active) [inline files/zh-cn/web/css/_doublecolon_-moz-list-number/index.md:10]:
```suggestion
**`::-moz-list-number`** [CSS](/zh-CN/docs/Web/CSS) [伪元素](/zh-CN/docs/Web/CSS/Pseudo-elements)是用于表示有序列表（{{HTMLElement("ol")}}）中列表项（{{HTMLElement("li")}}）的标记（通常是数字）是 [Mozilla 扩展](/zh-CN/docs/Web/CSS/Mozilla_Extensions)。
```

File: files/zh-cn/web/css/_doublecolon_-moz-list-number/index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+title: "::-moz-list-number"
+slug: Web/CSS/::-moz-list-number
+l10n:
+  sourceCommit: c8b447485fd893d5511d88f592f5f3aec29a725b
+---
+
+{{CSSRef}}{{Non-standard_header}}
+
+**`::-moz-list-number`** [CSS](/zh-CN/docs/Web/CSS) [伪元素](/zh-CN/docs/Web/CSS/Pseudo-elements)是用于表示有序列表（{{HTMLElement("ol")}}）中列表项（{{HTMLElement("li")}}）的标记（通常是数字）是 [Mozilla 扩展](/zh-CN/docs/Web/CSS/Mozilla_Extensions)。
```
Comment (skyclouds2001) [inline files/zh-cn/web/css/_doublecolon_-moz-list-number/index.md:10]:
```suggestion
**`::-moz-list-number`** [CSS](/zh-CN/docs/Web/CSS) [伪元素](/zh-CN/docs/Web/CSS/Pseudo-elements)是用于表示有序列表（{{HTMLElement("ol")}}）中列表项（{{HTMLElement("li")}}）的标记（通常是数字）的 [Mozilla 扩展](/zh-CN/docs/Web/CSS/Mozilla_Extensions)。
```

File: files/zh-cn/web/css/_doublecolon_-moz-list-number/index.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+{{EmbedLiveSample("示例")}}
+
+## 规范
+
+不属于任何规范。
+
+## 参见
+
+- {{cssxref("::-moz-list-bullet")}}
+- {{cssxref("::marker")}}
```
Comment (skyclouds2001) [inline files/zh-cn/web/css/_doublecolon_-moz-list-number/index.md:52]: 也连同原文少了个浏览器兼容性章节

Comment (yin1999) [reply files/zh-cn/web/css/_doublecolon_-moz-list-number/index.md:52]: @T34-active 补上啊，愣着干嘛 :(

File: files/zh-cn/web/css/_doublecolon_-moz-list-number/index.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
+---
+title: "::-moz-list-number"
+slug: Web/CSS/::-moz-list-number
+l10n:
+  sourceCommit: c8b447485fd893d5511d88f592f5f3aec29a725b
```
Comment (yin1999) [inline files/zh-cn/web/css/_doublecolon_-moz-list-number/index.md:5]:
```suggestion
  sourceCommit: 469d7dd1064fb2911aca110e1d653e19f3438ca5
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**